docs/deployment.md

```diff
@@ -0,0 +1,98 @@
+# Deploying OBMd
+
+## Download OBMd
+
+1. Download the OBMd binary from github. This downloads v0.2 of obmd.
+
+`$ wget https://github.com/CCI-MOC/obmd/releases/download/v0.2/obmd`
+
+Alternatively, you can clone the source code and build the binary yourself.
+
+2. Make it executable and place it in `/bin`.
```

Comment:
/usr/bin or /usr/local/bin would be a more standard location; ususally /bin is for stuff that is required early in the boot sequence. Typically stuff from distro packages goes in /usr/bin, and stuff not packaged (installed by the administrator) goes in /usr/local/bin.

Reply:
Ah, I see. At least in centos7 `/bin` is a symlink to `/usr/bin`. In ubuntu18.04, `/bin` is a regular directory. 
I'll change it to `/usr/local/bin` because that seems more appropriate.